### ISSUE TITLE
Rare diseases

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#447]
Background: EFO agrees that representation of rare diseases is unsatisfactory; MONDO has already taken steps to rectify the situation (https://github.com/monarch-initiative/mondo/issues/254). EFO may want to adopt similar solutions (or align to MONDO altogether, to minimize inconsistencies that need time and effort to fix in EFO3).
Open Targets already showed interest in an advanced solution (generate disease prevalence statements, #9). 
I will add more details.

**Comment by paolaroncaglia** [comment]
@cmungall 
We're aware of this work in MONDO on obsoletion of rare-X terms: 
https://github.com/monarch-initiative/mondo/issues/254
That ticket is closed. However, we see that there are still some 'rare disease' terms in MONDO, see my comment above. Could you please explain what strategy you followed - e.g. did you start from rare genetic diseases first, and are leaving other 'rare' terms for later? We'd like to align on this work, not last because any significant change in MONDO will result in EFO errors after your release :-) Many thanks in advance!

**Comment by zoependlington** [comment]
@paolaroncaglia To answer:
> 'rare' is already used in EFO3, but I assume that all its class usage cases in Protege (~900 of them) are for MONDO terms. @zoependlington could you confirm please?

I can confirm all uses of 'rare' are in MONDO terms or EFO terms that have been mapped to MONDO terms and therefore inherited the 'rare' from the MONDO mapping.

**Comment by cmungall** [comment]
We obsoleted the ones for which there was a clear replacement, Rare X -->
X, i.e. where we trivially shadow an existing class with a rare subclass

So ones like "rare disease with dentinogenesis imperfecta" remain, because
we have no "disease with dentinogenesis imperfecta" (such a disease would
already be rare I imagine)

I think we need to discuss these kinds of grouping classes (e.g. X with Y)
separately. I think many could be obsoleted but this would be done in a
very conservative way, in contrast to the Rare X --> X case.

**Comment by simonjupp** [comment]
Paola, can we compile a list of all the Rare X -> X that would be candidates for removal from EFO? We should check internally if any of these have been used for direct annotation and ensure there's a migration path for the downstream consumers.

**Comment by paolaroncaglia** [comment]
@cmungall thanks for explaining.
@simonjupp I opened a ticket for that task, assigned it to me and linked it to this epic.

**Comment by paolaroncaglia** [comment]
Update on obsoleting (some) Orphanet 'rare x' diseases in EFO: part of this task will overlap with cleaning up undesired Orphanet-derived superclasses (Epic https://github.com/EBISPOT/efo/issues/322). That work is now scheduled for the first quarter of 2020 as it's top priority for Open targets.

**Comment by paolaroncaglia** [comment]
Update: we are still working on this in collaboration with Mondo; might be addressed in the second quarter of 2020 as part of the Orphanet work.

**Comment by paolaroncaglia** [comment]
Update: we're working with Mondo to address 'rare disease' terms that are still in Mondo and that Mondo already tagged as obsoletion candidates. Those are Orphanet-derived terms, so addressing them also addresses some Orphanet-derived parentage issues. In detail, Nicole and I have been working on [this spreadsheet](https://docs.google.com/spreadsheets/d/1pASj2MnBI285ogOeKVqEVe2TsKCvOWDFFNq_ekN2juw/edit#gid=0). Making a note here to 
- [ ] Keep an eye on Mondo tickets labelled "EFO" and where title contains "rare". When Mondo has resolved on actions, see if EFO needs to edit those terms too.

**Comment by paolaroncaglia** [comment]
@zoependlington I haven't worked on disease terms since the time of my latest comment, but I know that Mondo has progressed steadily with its cleanup of 'rare disease' terms. The results of their work-in-progress should then all percolate to EFO eventually. Is this ticket still needed, and if yes, should it be updated?

**Comment by zoependlington** [comment]
Yes, any remaining rare terms should be Orphanet terms that will eventually be obsoleted/replaced by Mondo terms. There may be a few EFO terms that need looking into however, so this ticket should remain open.

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26).** This epic (opened 2019) asked EFO to sort out its rare-disease representation in line with MONDO's "Rare X → X" obsoletion strategy (monarch-initiative/mondo#254). A status check of where things stand:

**What's done / tracked elsewhere:**
- Most checklist items in the description are complete (structure surveys, MONDO's `has modifier some 'rare'` pattern, confirmation that 'rare' usage in EFO traces to MONDO).
- The "Rare X → X" candidate listing was spun out to #452 (still open, backlog).
- Prevalence statements were tracked in #9, closed in 2020.
- The broader Orphanet cleanup this epic depends on is tracked by epic #322 (is-a overloading / MONDO replacement) and #673 (Orphanet terms with no MONDO mapping): of the 6,213 Orphanet classes native to efo-edit.owl, 4,180 are now obsoleted with MONDO replacements and 2,033 remain active.
- The use-case of *identifying* rare diseases in EFO is now restated in #1695.

**What remains:** efo-edit.owl still contains 96 active classes labelled "Rare …" — 94 Orphanet grouping classes (e.g. Orphanet:156601 "Rare genetic hepatic disease", Orphanet:167762 "Rare disease with dentinogenesis imperfecta") plus 2 native EFO terms (EFO:0022487 "rare syndromic intellectual disability", EFO:0022986 "rare venous malformation"). Their fate is determined by the #322/#673 obsoletion pipeline rather than by anything specific to this ticket.

**Recommendation:** close this epic as superseded — its remaining actionable work is fully covered by #322, #673, #452, and #1695 — or, if an umbrella is still wanted, update the description to point at those four tickets as the live workstreams.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
